app/models/submission.rb

```diff
@@ -70,4 +70,20 @@ def eligible_for_evaluation?
   def selected_to_advance?
     winner?
   end
+
+  def eligibility_checkbox_disabled?
+    evaluators.any?
+  end
```

Comment:
I tried to unassign an evaluator from the submission but this query still returns true and the checkbox is not re-enabled. it should filter for the evaluators where assignment status in ['assigned', 'recused'] so we don't flag 'unassigned' or 'recused_unassigned' as being evaluators.
```suggestion
  def eligibility_checkbox_disabled?
    evaluators.where(status: [:assigned, :recused]).exists?
  end
```